grpc-sys/build.rs

```diff
@@ -141,7 +138,7 @@ fn build_grpc(cc: &mut Build, library: &str) {
         }
         // Uses zlib from libz-sys.
         setup_libz(&mut config);
-        config.build_target(library).uses_cxx11().build()
```

Comment:
Why removing this?

Reply:
That is introduced 3y ago. Now mac CI works fine and I think it should be removed if there is no special reason.